hotdoc/extensions/gi/utils.py

```diff
@@ -200,16 +202,17 @@ def get_language_classes():
     Banana banana
     """
     all_classes = {}
-    deps_map = {}
 
-    for entry_point in pkg_resources.iter_entry_points(
-            group='hotdoc.extensions.gi.languages', name='get_language_classes'):
+    for entry_point in entry_points(
+            group='hotdoc.extensions.gi.languages',
+            name='get_language_classes'):
         try:
+            ep_name = entry_point.name
             activation_function = entry_point.load()
             classes = activation_function()
         # pylint: disable=broad-except
         except Exception as exc:
-            info("Failed to load %s" % entry_point.module_name, exc)
+            info("Failed to load %s" % str(entry_point), exc)
```

Comment:
By the same token, `entry_point.name` (the equivalent of `entry_point.module_name` from `pkg_resources` isn't necessarily going to exist on every result from `entry_points()`, so we can't rely on it for logging. Logging the string value of `entry_point` ensures that it'll be logged even if it isn't a valid `EntryPoint` object.